src/connectors/traderjoe/traderjoe.ts

```diff
@@ -9,19 +9,22 @@ import {
 } from 'ethers';
 import { isFractionString } from '../../services/validators';
 import { TraderjoeConfig } from './traderjoe.config';
-import routerAbi from './IJoeRouter02.json';
-import {
-  Fetcher,
-  Percent,
-  Router,
-  Token,
-  TokenAmount,
-  Trade,
-  Pair,
-} from '@traderjoe-xyz/sdk';
 import { logger } from '../../services/logger';
 import { Avalanche } from '../../chains/avalanche/avalanche';
 import { ExpectedTrade, Uniswapish } from '../../services/common-interfaces';
+import { Token, TokenAmount, JSBI, Percent } from '@traderjoe-xyz/sdk';
+import {
+  LBRouterV21ABI,
+  PairV2,
+  RouteV2,
+  TradeV2,
+} from '@traderjoe-xyz/sdk-v2';
+import { EVMTxBroadcaster } from '../../chains/ethereum/evm.broadcaster';
+import { createPublicClient, http } from 'viem';
+import { avalanche, avalancheFuji } from 'viem/chains';
+
+const MAX_HOPS = 2;
```

Comment:
would users need to modify this parameter? if so, I think it should be a param in `traderjoe.yml`

Reply:
It's not a parameter regular users should modify in this case. Changing that value without adding more path tokens would have no effect on the final route. It would be a different case if there is an auto/smart router.